__tests__/index.test.js

```diff
@@ -218,4 +218,64 @@ describe('/api/articles/:article_id/comments', () => {
         expect(response.body.msg).toBe('Bad request');
       });
   });
+})
+
+describe('/api/articles/:article_id', () => {
+  test('PATCH:201 inserts a new comment to the comments table and sends the new comment back to the client', () => {
+    const patchObj = {
+      inc_votes : 100
+    };
+    return request(app)
+      .patch('/api/articles/3')
+      .send(patchObj)
+      .expect(200)
+      .then((response) => {
+        expect(response.body.article.length).toBe(1);
+        const article = response.body.article[0];
```

Comment:
You shouldnt not need use the 0 index to access the article from body like this we should have access to it directly from .article